cpp/src/slice2cpp/CPlusPlusUtil.cpp

```diff
@@ -510,7 +505,7 @@ Slice::typeToString(
     ClassDeclPtr cl = dynamic_pointer_cast<ClassDecl>(type);
     if (cl)
     {
-        return "::std::shared_ptr<" + getUnqualified(cl->scoped(), scope) + ">";
+        return getUnqualified(cl->scoped(), scope) + "Ptr";
```

Comment:
I updated the generated code to use/generate XxxPtr and not ::std::shared_ptr<Xxx>.